File: 2.0/java/media/getMediaData.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.callstexts.model.MediaType;

import java.io.InputStream;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        InputStream is = client.mediaApi().getData(617067920003L, MediaType.MP3);
    }
}

File: 2.0/csharp/media/getMediaData.cs
using System.IO;
using CallfireApiClient;
using CallfireApiClient.Api.CallsTexts.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        Stream stream = client.MediaApi.GetData(379506003, MediaType.MP3);
    }
}
